Pinta.Core/Managers/ImageConverterManager.cs

```diff
@@ -51,7 +51,7 @@ private static IEnumerable<FormatDescriptor> GetInitialFormats ()
 		// Create all the formats we have our own importers/exporters for
 
 		OraFormat oraHandler = new ();
-		FormatDescriptor oraFormatDescriptor = new ("OpenRaster", new[] { "ora", "ORA" }, new[] { "image/openraster" }, oraHandler, oraHandler);
+		FormatDescriptor oraFormatDescriptor = new ("OpenRaster", new[] { "ora", "ORA" }, new[] { "image/openraster" }, oraHandler, oraHandler, true);
```

Comment:
Minor style suggestion - use a named argument  e.g. `supportsLayers: true` so that the meaning of `true` or `false` is more obvious to readers of this file